Assets/Scripts/UI/OpenURLInBrowser.cs

```diff
@@ -0,0 +1,14 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Netherlands3D.Twin
+{
+    public class OpenURLInBrowser : MonoBehaviour
```

Comment:
Ik zou er zelf denk ik een Button behaviour van gemaakt die zichzelf op de onclick registreert. Dit om te voorkomen dat we rondslingerende utility componenten hebben en het makkelijker is om te herleiden of zelfs op een prefab te zetten

Reply:
En dus dat de URL ook een serialized field is :)

Reply:
Dat is ook hoe hij bedoeld is, alleen bepaal je nu niet van tevoren het type event ( en zou je evt. ook een andere event handler kunnen gebruiken ).

Hier is hoe hij nu op de knop staat:

![image](https://github.com/Netherlands3D/twin/assets/11850024/5bc94a5b-fdf1-4f45-8f8d-e847248b4c32)